pr-data.csv

```diff
@@ -4369,7 +4369,7 @@ https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9
 https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.BigQueryConvertersTest.testFailsafeJsonToTableRowValidInput,ID,MovedOrRenamed,,https://github.com/GoogleCloudPlatform/DataflowTemplates/commit/0d01accc1cfa29b762a764bc226496bfde25165c#diff-8127053753c286e4275e5e1d0f1f737b7ee24a3e6d41c1feede58ab5282eae19
 https://github.com/GoogleCloudPlatform/DataflowTemplates,cc17ed154d8c3c385ab3f5a3653a6e4c02539bfa,v1,com.google.cloud.teleport.templates.common.BigQueryConvertersTest.testFailsafeJsonToTableRowValidInput,ID,,,
 https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckNoKeyAllInvalid,ID,,,
-https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckNoKeyBothCorrectAndInvalid,ID,,,
+https://github.com/GoogleCloudPlatform/DataflowTemplates,0d01accc1cfa29b762a764bc226496bfde25165c,v1,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckNoKeyBothCorrectAndInvalid,ID,Opened,https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1919,
```

Comment:
Please do not change the commit SHA where the flaky was reported. What's the new SHA that you want to add? Maybe you can report in between that the test was moved. You could get +2 points for that (but need to find the commit where the test was moved).

Reply:
I removed the change of this line and will figure out the correct format to create a new PR. Sorry for the confusion.